website/docs/reference/snapshot-configs.md

```diff
@@ -222,106 +252,141 @@ snapshots:
 
 ```
 
+</VersionBlock>
+
 </TabItem>
 
 </Tabs>
 
-
 ## Configuring snapshots
-Snapshots can be configured in one of three ways:
-
-1. Using a `config` block within a snapshot
-2. Using a `config` [resource property](/reference/model-properties) in a `.yml` file
-3. From the `dbt_project.yml` file, under the `snapshots:` key. To apply a configuration to a snapshot, or directory of snapshots, define the resource path as nested dictionary keys.
-
-Snapshot configurations are applied hierarchically in the order above.
-
-### Examples
-#### Apply configurations to all snapshots
-To apply a configuration to all snapshots, including those in any installed [packages](/docs/build/packages), nest the configuration directly under the `snapshots` key:
-
-<File name='dbt_project.yml'>
-
-```yml
+Snapshots can be configured in multiple ways:
 
-snapshots:
-  +unique_key: id
-```
-
-</File>
+<VersionBlock firstVersion="1.9">
 
+1. Defined in YAML files using a `config` [resource property](/reference/model-properties), typically in your [snapshots directory](/reference/project-configs/snapshot-paths) (available in [Versionless](/docs/dbt-versions/versionless-cloud) or and dbt Core v1.9 and higher).
+2. From the `dbt_project.yml` file, under the `snapshots:` key. To apply a configuration to a snapshot, or directory of snapshots, define the resource path as nested dictionary keys.
+</VersionBlock>
 
-#### Apply configurations to all snapshots in your project
-To apply a configuration to all snapshots in your project only (for example, _excluding_ any snapshots in installed packages), provide your project name as part of the resource path.
+<VersionBlock lastVersion="1.8">
 
-For a project named `jaffle_shop`:
+1. Defined in YAML files using a `config` [resource property](/reference/model-properties), typically in your [snapshots directory](/reference/project-configs/snapshot-paths) (available in [Versionless](/docs/dbt-versions/versionless-cloud) or and dbt Core v1.9 and higher).
+2. Using a `config` block within a snapshot
+3. From the `dbt_project.yml` file, under the `snapshots:` key. To apply a configuration to a snapshot, or directory of snapshots, define the resource path as nested dictionary keys.
 
-<File name='dbt_project.yml'>
+Note that in Versionless and dbt v1.9 and later, snapshots are defined in an updated syntax using a YAML file within your `snapshots/` directory (as defined by the [`snapshot-paths` config](/reference/project-configs/snapshot-paths)). For faster and more efficient management, consider the updated snapshot YAML syntax, [available in Versionless](/docs/dbt-versions/versionless-cloud) or [dbt Core v1.9 and later](/docs/dbt-versions/core).
 
-```yml
+</VersionBlock>
 
-snapshots:
-  jaffle_shop:
-    +unique_key: id
-```
+Snapshot configurations are applied hierarchically in the order above.
```

Comment:
We [don't yet](https://github.com/dbt-labs/docs.getdbt.com/issues/4371) have standardized language and style that communicates the precedence order. In the meantime, maybe a slight modifier here to clarify:
```suggestion
Snapshot configurations are applied hierarchically in the order above with higher taking precedence.
```

I just now added snapshot configs to this list: https://github.com/dbt-labs/docs.getdbt.com/issues/4371